ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -157,6 +159,44 @@ public String getVersion() {
         return CompletableFuture.completedFuture(buildRaoResult(bestScore, preOptimVariant, bestResultVariant));
     }
 
+    private SystematicSensitivityAnalysisResult runSensi(Network network,
+                                                         Crac crac,
+                                                         ComputationManager computationManager,
+                                                         SensitivityComputationParameters parameters) {
+        SystematicSensitivityAnalysisResult sensitivityAnalysisResult = SystematicSensitivityAnalysisService.runAnalysis(network, crac, computationManager, parameters);
+        // Failure if some sensitivities are not computed
+        if (sensitivityAnalysisResult.getStateSensiMap().containsValue(null) || sensitivityAnalysisResult.getCnecFlowMap().isEmpty()) {
+            throw new SensitivityComputationException(String.format("Sensitivity computation failed with %s sensitivity parameters.", useFallbackSensiParams ? "fallback" : "default"));
+        } else {
+            return sensitivityAnalysisResult;
+        }
+    }
+
+    private SystematicSensitivityAnalysisResult runWithParamSelection(Network network,
```

Comment:
runWithSensitivityParameterSelection ? I think it's better to have a longer but more explicit name.

Reply:
Or runSystematicSensitivityAnalysisWithFallback to be consistent again

Reply:
Las time, you told me:
> Rename runSensiWithFallback ? When I read I felt like this was only the fallback calculation but it's not
Isn't it a problem anymore to have "WithFallback" in the method name?

Reply:
WithParametersSwitch ?

Reply:
ok